.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- This issue tracker is only for technical issues related to Dimecoin Core.
+
+General bitcoin questions and/or support requests are best directed to the dedicated support channels for Dimecoin at https://t.me/dimeofficial support or contacting via [email](mailto:support@dimecoinnetwork.com) 
+
+For reporting security issues, please email developer@dimecoinnewtork.com.
```

Comment:
same comment as before, probably shouldn't include email here.

Reply:
amended contact information to avoid spam